service/interfaces.go

```diff
@@ -31,3 +33,9 @@ type HealthChecker interface {
 	Stop()
 	AddCheck(name string, checker healthcheck.Checker) (err error)
 }
+
+//Mongoer defines the required methods from Mongo
+type Mongoer interface {
```

Comment:
Using the -er suffix is generally used for single function interfaces. I think it's good to name interfaces for their functional purpose - in this case something like `PermissionsStore`?